src/OSExtensions/OSExtensions.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net461;netstandard1.6</TargetFrameworks>
```

Comment:
Because OSExtensions is its own DLL (and a pain to update) and probably does not use APIs that are different between net45 and netStandard1.6 we should leave this as net45.   This allows us to move to net45 without having to update OSExtensions in the future.